### PR TITLE
Fix 'Actions' blocks appearance

### PR DESCRIPTION
**Pull request by obniavko**
dev
## ZeroWaste project

* [Project ticket #756](https://github.com/ita-social-projects/ZeroWaste/issues/756)
* [Project ticket #757](https://github.com/ita-social-projects/ZeroWaste/issues/757)

## Code reviewers

- [x] @kisiohlova 
- [ ] @loqimean 

## Summary of issue

1. 'Actions' icons are combined into one block on 'Calculators' and 'Categories' pages.
2. 'Show more' icon has black color and is not aligned in regards of the title

## Summary of change

 1. Divided  'Actions' blocks into the separate columns.

Calculators

Users

Products

Categories

 2. Aligned 'Show more' icon and made it green.

Messages

## CHECK LIST
- [x]  СI passed
- [x]  Сode coverage >=95%
- [x]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [x]  I've checked new feature as logged in and logged out user if needed
- [x]  PR meets all conventions

### CODE REVIEW

**Review by kisiohlova — APPROVED** [review]

File: app/views/account/users/index.html.erb [hunk 1]
```diff
@@ -9,42 +9,40 @@
...
+        <tr id="user-info-<%= user.id %>">
+          <td><%= user.email %></td>
+          <td><%= user.first_name %></td>
+          <td><%= user.last_name %></td>
+          <td><%= user.last_sign_in_at %></td>
+          <td class="text-center"><%= link_to icon('fa-solid', 'eye'), account_user_path(id: user.id) %></td>
+          <td class="text-center"><%= link_to icon('fa-solid', 'edit'), edit_account_user_path(id: user.id) %></td>
+          <td class="text-center">
+            <%= link_to icon('fa-solid', user.blocked? ? 'lock' : 'lock-open'),
+                        account_user_path(id: user, user: { blocked: user.blocked? ? false : true }),
```
Comment (loqimean) [inline app/views/account/users/index.html.erb:35]: це краще мати в окрмому методі, але то в окремому ПР-і тре пофіксити

Comment (loqimean) [reply app/views/account/users/index.html.erb:35]: та і наче на це вже є хелпер

File: app/views/account/users/index.html.erb [hunk 2]
```diff
@@ -9,42 +9,40 @@
...
+                        onclick: "alert('Are you sure?')", method: :patch %>
```
Comment (loqimean) [inline app/views/account/users/index.html.erb:36]: не гуд так робити, у нас для того є турбо конфірм

File: app/views/account/users/index.html.erb [hunk 3]
```diff
@@ -9,42 +9,40 @@
...
+          </td>
+          <td class="text-center">
+            <%= link_to account_user_path(id: user), method: :delete,
+                        data: { confirm: t('.confirm_delete') } do %>
```
Comment (loqimean) [inline app/views/account/users/index.html.erb:40]: а от цей конфірм не працює

**Pull request merged**